### PR TITLE
Update cypress guidelines

### PR DESCRIPTION
**Pull request by nixocio**
Update cypress guidelines

### CODE REVIEW

File: cypress/CYPRESS.md [hunk 1]
```diff
@@ -158,15 +163,16 @@ End-to-End tests for our project are located in the `cypress/e2e` directory. The
...
+Many AWX resources have an organization, and if the organization is deleted, the resource gets deleted. By creating an organization at the start of the tests and using it with resources, then deleting the organization, most resources clean up.
 
-There are a few cases where resources do not get cleaned up, such as when a project is syncing, deleting the organization leaves the project around without an associated organization.
+There are a few cases where resources do not get cleaned up, such as when a project is syncing. Deleting the organization leaves the project around without an associated organization.
 
-To handle this, the tests delete E2E organizations older than 2 hours old. Then the other tests delete resources that do not have an organization. This makes the system eventually cleanup the E2E resources.
+To handle this, the tests delete E2E organizations older than 2 hours. Then the other tests delete resources that do not have an organization. This makes the system eventually clean up the E2E resources.
```
Comment (nixocio) [inline cypress/CYPRESS.md:178]: I do think this statement is true anymore, this one should be revisited.

Comment (vishsanghishetty) [reply cypress/CYPRESS.md:178]: not sure, I wasn't aware of this
@vidyanambiar can help us with this

**Review by vishsanghishetty — APPROVED** [review]

File: cypress/CYPRESS.md (hunk 1 above)
Comment (nixocio) [reply cypress/CYPRESS.md:178]: @vishsanghishetty if that is no longer true, we can add a follow up commit.

Comment (vishsanghishetty) [reply cypress/CYPRESS.md:178]: sounds good ✅

**Pull request merged**